tracer/src/Datadog.Trace/Tags.cs

```diff
@@ -417,6 +430,21 @@ public static class Tags
         /// </summary>
         internal const string AppSecEvent = "appsec.event";
 
+        /// <summary>
+        /// The details of the security event
+        /// </summary>
+        internal const string AppSecJson = "_dd.appsec.json";
+
+        /// <summary>
+        /// Should contain the public IP of the host initiating the request.
+        /// </summary>
+        public const string ActorIp = "actor.ip";
+
+        /// <summary>
+        /// The ip as reported by the framework.
+        /// </summary>
+        public const string NetworkClientIp = "network.client.ip";
+
```

Comment:
If these are appsec-specific, maybe we should make them internal? `ActorIp` naming seems suspect for example (though I imagine, has already been decided from elsewhere). Smacks of actor frameworks like Orleans/Akka etc to me, so should probably not be public IMO.

Reply:
Yes, it's not particularly clear to me why something in this class are public while others are internal. However, I agree internal is probably safer, as it's much easier to move things internal -> public rather than the other way round. Fixed in f4df71c6

Reply:
> it's not particularly clear to me why something in this class are public while others are internal

Historically, they were all `public` just by default, but in 2.0 we moved tags that are set by automatic integrations to be internal, as they're not intended for users to set them in manual spans (Obviously they can if they just re-use the string, but we shouldn't "suggest" them I think)